### DISCUSSION TITLE
candidate source tracking

### DISCUSSION THREAD

**Discussion by otaviogit** [bullhorn/career-portal#416] [Q&A] [answered]
Sorry newbie here!

We are a job board and we're scraping our client's bullhorn jobs from their Wordpress careers site but when we send the 'candidate source tracking code', their ATS dashboard does not track incoming candidates that we send them. 

Bullhorn support told the client that the 'candidate source' is not being relayed to the Bullhorn ATS from their corporate site.

Is there a way to get their jobs listing from bullhorn so we can send them directly to the ATS, not the client's Wordpress?

Would that be a jobs portal?   If yes, does one exist by default or does client have to set it up?

help!

**Comment by charlesabarnes** [comment, marked as answer]
Hello

When the jobs are scraped, how are the sources being added?  Also what content is being scraped? 

If you link to a specific job with the `source`parameter added in the url, it'll reflect in the system. 

You can also leverage the API as an option

**Reply by otaviogit** [reply]
Hi Charles,

- we scrape the clients' front-facing website job postings (title, description, etc) and cross-post them to our network of job boards.

- When candidates click 'Apply Now' on our sites. they're sent to the original job posting URL on clients site (run in wordpress) AND we append that URL with a parameter that we have used in the past w/ other Bullhorn clients.

However, bullhorn tells us that parameter is not being passed from client site to the ATS.

example:
https://diversityjobs.com/jobsearch/display/728097824 

click 'apply now' (TIP: you may right-click and choose new tab to skip that email collection modal dialog)

you'll be sent to :  https://www.liongard.com/wp-content/plugins/bullhorn-oscp/?source=DiversityJobs&src_divjobs=1628264689&divkey=9fbf3d83b503e8a1e0c28310aac096a1&utm_campaign=diversityjobs&utm_source=diversityjobs#/jobs/16

whereas source=DiversityJobs is the BH code we use.

so i wonder how can we skip the clients' WordPress site.

Thank you

**Reply by charlesabarnes** [reply]
Hello, this is because the parameters are being added before the job route information `#/jobs/16`  If it is added after, it should work.   EX
https://www.liongard.com/wp-content/plugins/bullhorn-oscp/#/jobs/16?source=DiversityJobs&src_divjobs=1628264689&divkey=9fbf3d83b503e8a1e0c28310aac096a1&utm_campaign=diversityjobs&utm_source=diversityjobs

Also you can circumvent the portal by leveraging the Bullhorn API.  You can read more about the API here http://bullhorn.github.io/rest-api-docs/

**Reply by otaviogit** [reply]
bingo! thank you!!!
